pkg/kubeutils/kubeutils.go

```diff
@@ -318,6 +318,29 @@ func WaitForControllerNode(ctx context.Context, kcli client.Client, name string)
 	return nil
 }
 
+// WaitForJob waits for a job to be ready.
+func WaitForJob(ctx context.Context, cli client.Client, ns, name string, maxSteps int, completions int32) error {
```

Comment:
i might be missing it, but where is this used?

Reply:
It was initially added when I was having the binary wait for the job to complete
it's not used now